src/NUnitEngine/nunit.engine/Services/ExtensionService.cs

```diff
@@ -98,32 +97,20 @@ public void EnableExtension(string typeName, bool enabled)
         #region Public Methods - Extension Points
 
         /// <summary>
-        /// Get an ExtensionPoint based on its unique identifying path.
+        /// Get an extension point based on its unique identifying path.
         /// </summary>
         public ExtensionPoint GetExtensionPoint(string path)
         {
             return _pathIndex.ContainsKey(path) ? _pathIndex[path] : null;
         }
 
         /// <summary>
-        /// Get an ExtensionPoint based on the required Type for extensions.
+        /// Gets the extension point with the specified full type name.
         /// </summary>
-        public ExtensionPoint GetExtensionPoint(Type type)
+        public ExtensionPoint GetExtensionPointFromTypeName(string fullName)
```

Comment:
Public API shape change for `nunit.engine`
(`internal`? Do we care about public shape changes in this assembly?)